Fst2List.cpp

```diff
@@ -1979,18 +1983,39 @@ int CFstApp::outWordsOfGraph(int depth) {
   return 0;
 }
 
-//
 //
 //
 //
 
-// TODO(jhondoe) Use UnitexGetOpt instead argc, argv
+const char* optstring_Fst2List=":o:Sp:a:t:l:i:mdf:vVhs:r:c:";
+const struct option_TS lopts_Fst2List[]= {
+  {"output",required_argument_TS,NULL,'o'},
+  {"automaton",required_argument_TS,NULL,'a'},
```

Comment:
Important, this option is not useful because now `-a` and `-t` are mutually exclusive. If `-a` represents the default behavior, the user can modify it by simple passing `-t`